fabric-resource-loader-v0/src/main/resources/fabric-resource-loader-v0.accesswidener

```diff
@@ -0,0 +1,3 @@
+accessWidener	v2	named
+
+accessible    method    net/minecraft/resource/NamespaceResourceManager$class_7083	<init>	(Lnet/minecraft/resource/NamespaceResourceManager;Lnet/minecraft/util/Identifier;Lnet/minecraft/util/Identifier;Lnet/minecraft/resource/ResourcePack;)V
```

Comment:
This file probably should be removed as it's not actually used in the quilted module.